docs/releasing/publishing.md

```diff
@@ -59,43 +66,44 @@
   npm link ../govuk-frontend/package/
   ```
 
-  When you have finished you need to unlink the package
+  When you have finished, you need to unlink the package.
 
   ```bash
   npm unlink ../govuk-frontend/package/
   ```
 
 10. Create a pull request and copy the changelog text.
-   When reviewing the PR, check that the version numbers have been updated and that the compiled assets use this version number.
+   When reviewing the PR, check that the version-numbers have been updated and that the compiled assets use this version-number.
 
-11. Once the pull request is approved, merge to **master**.
+11. Once a reviewer approves the pull request, merge it to **master**.
 
-12. Checkout **master** and pull the latest changes.
+12. Checkout the **master** branch and pull the latest changes.
 
 13. Log into npm (`npm login`), using team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/master/npm/govuk-patterns-and-tools).
 
-14. Run `npm run publish-release`, you will be prompted to continue or cancel.
+14. Run `npm run publish-release`. You will now be prompted to continue or cancel.
 
-15. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases)
+15. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases) as follows:
   - select the latest tag
   - press 'Edit tag'
-  - set "GOV.UK Frontend v[version-number]" as the title
+  - set 'GOV.UK Frontend v[version-number]' as the title
   - add release notes from changelog
   - attach the generated ZIP that has been generated at the root of this project
   - publish release
 
-16. Log out from npm
-```bash
-npm logout
-```
+16. Run `npm logout` to log out from npm.
 
-# Steps you must complete after you release a new version of GOV.UK Frontend
+# After you release a new version of GOV.UK Frontend
 
-1. Update the GOV.UK Design System, Prototype Kit, and Frontend Docs to use a new release of GOV.UK Frontend, and release a new version of the Prototype Kit.
+1. Update the GOV.UK Design System, Prototype Kit and Frontend Docs to:
+  - use a new release of GOV.UK Frontend
+  - release a new version of the Prototype Kit
 
-2. Post your summary of the:
-- GOV.UK Frontend release in the govuk-design-system channels on [GDS Slack](https://gds.slack.com/archives/CAF8JA25U) and [x-gov Slack](https://ukgovernmentdigital.slack.com/archives/C6DMEH5R6)
-- Prototype kit release in the [prototype-kit channel](https://ukgovernmentdigital.slack.com/archives/C0647LW4R) on x-gov Slack
+2. Post announcements of the release to:
+  - GOV.UK Frontend in the govuk-design-system channels on [GDS Slack](https://gds.slack.com/archives/CAF8JA25U) and [x-gov Slack](https://ukgovernmentdigital.slack.com/archives/C6DMEH5R6)
```

Comment:
Could we use the https://ukgovernmentdigital.slack.com/app_redirect?channel=govuk-design-system etc. syntax for linking to Slack channels? They're more readable and less likely to break in the future than the archive links.